A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js

```diff
@@ -116,6 +120,19 @@ class A320_Neo_PFD_MainPage extends NavSystemPage {
         if (!this.hasInitialized) {
             return;
         }
+
+        if (this.yokePositionThrottler.canUpdate(_deltaTime) != -1) {
+            const YokeXPosition = 40.45 + (18.4 * SimVar.GetSimVarValue("YOKE X POSITION", "Position"));
+            const YokeYPosition = 47.95 - (14.45 * SimVar.GetSimVarValue("YOKE Y POSITION", "Position"));
+
+            this.groundCursor.style.left = YokeXPosition.toString() + "%";
+            this.groundCursor.style.top = YokeYPosition.toString() + "%";
+        }
+
+        _deltaTime = this.updateThrottler.canUpdate(_deltaTime);
```

Comment:
This will need a let _deltaTime at the start of this function.

Reply:
done

Reply:
at the very first line of onUpdate() function, const _deltaTime declaration needs to be let _deltaTime

Reply:
Yes, that's done in the latest version.